frontend/src/metabase/search/components/SearchResult/components/ItemIcon.styled.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "@emotion/styled";
+import type { SearchModelType } from "metabase-types/api";
+import { color, lighten } from "metabase/lib/colors";
+
+function getColorForIconWrapper({
+  active,
+  type,
+}: {
+  active: boolean;
+  type: SearchModelType;
+}) {
+  if (!active) {
+    return color("text-medium");
+  }
+  if (type === "collection") {
+    return lighten("brand", 0.35);
+  }
+  return color("brand");
+}
+
+export const IconWrapper = styled.div<{
```

Comment:
`getColorForIconWrapper` & `IconWrapper` should be removed from `SearchResult.styled.tsx`